code/drasil-printers/Language/Drasil/Printing/Import.hs

```diff
@@ -97,6 +99,11 @@ processExpo a
 assocExpr :: P.Ops -> Int -> [Expr] -> PrintingInformation -> P.Expr
 assocExpr op prec exprs sm = P.Row $ intersperse (P.MO op) $ map (expr' sm prec) exprs
 
+dispExpr :: DisplayExpr -> PrintingInformation -> P.Expr
+dispExpr (AlgebraicExpr e) sm = expr e sm
+dispExpr (Defines l r)     sm = P.Row [dispExpr l sm, P.MO P.Eq, dispExpr r sm] -- TODO: Use new symbol?
```

Comment:
Do we want to replace the Equals symbol with a new symbol for `Defines`?

Reply:
Later PR. Let's minimize the changes in this one.

Reply:
Sounds good.